routes/web.php

```diff
@@ -43,9 +43,7 @@
 
         Route::resource('orders.items', 'OrderItemsController', ['only' => ['update']]);
 
-        Route::get('/', function () {
-            return ujs_redirect(route('admin.store.orders.index'));
-        });
+        Route::get('/', '\App\Http\Controllers\RedirectController')->name('redirect.admin.store.orders.index');
```

Comment:
`Route::currentRouteName()` returns a version where all punctuation are dots; the action of `RedirectController` broke with the colon because `explode()` didn't do anything